context/app/static/js/components/search/Facets/ExistsFacet.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+
+import { trackEvent } from 'js/helpers/trackers';
+import { ExistsValues, isExistsFilter, useSearchStore } from '../store';
+import { StyledCheckBoxBlankIcon, StyledCheckBoxIcon, StyledCheckbox, StyledFormControlLabel } from './style';
+import { useGetFieldLabel } from '../fieldConfigurations';
+import FacetAccordion from './FacetAccordion';
+
+function ExistsFacet({ filter, field }: { filter: ExistsValues; field: string }) {
+  const analyticsCategory = useSearchStore((state) => state.analyticsCategory);
+  const filterExists = useSearchStore((state) => state.filterExists);
+
+  const active = filter.values;
+  const getFieldLabel = useGetFieldLabel();
+  const fieldLabel = getFieldLabel(field);
+
+  const handleClick = useCallback(() => {
+    filterExists({ field });
+
+    const facetAction = active ? 'Unselect' : 'Select';
+    trackEvent({
+      category: analyticsCategory,
+      action: `${facetAction} Facet`,
+      label: field,
+    });
+  }, [active, analyticsCategory, filterExists, field]);
+
+  return (
+    <FacetAccordion title={fieldLabel} position="inner">
+      <StyledFormControlLabel
+        control={
+          <StyledCheckbox
+            checked={active}
+            name={`${fieldLabel}-checkbox`}
+            color="primary"
+            icon={<StyledCheckBoxBlankIcon />}
+            checkedIcon={<StyledCheckBoxIcon />}
+            onChange={handleClick}
+          />
+        }
+        label="True"
+      />
+    </FacetAccordion>
```

Comment:
This works, but I wonder if it could be cleaner if we have this be a standalone facet without an accordion and use the field label as the checkbox label?